common/company_types/00_companies_italy.txt

```diff
@@ -68,8 +68,8 @@ company_ansaldo = {
 	}
 	
 	prosperity_modifier = {
-		unit_navy_offense_mult = 0.05
-		unit_army_offense_mult = 0.1
+		building_power_plant_throughput_add = 0.1
+		interest_group_ig_industrialist_approval_add = 2
```

Comment:
"Until the end of the 19th century, the company focused on manufacturing and repairing railway components, quickly becoming a 10,000-worker company with seven factories, and starting to expand into sectors such as shipbuilding and mechanical works in general."

The power thing didnt happen until 1980 I think? Looking at their wiki article.

[https://en.wikipedia.org/wiki/Gio._Ansaldo_%26_C.](https://en.wikipedia.org/wiki/Gio._Ansaldo_%26_C.)

Reply:
Hm, my bad, i'll look into what else I can use. It's hard for some of these to get it right